packages/neuron-ui/src/containers/Navbar/index.tsx

```diff
@@ -107,6 +112,10 @@ const Navbar = () => {
     }
   }, [network?.readonly])
 
+  const gotoCompatile = useCallback(() => {
+    openExternal(`https://neuron.magickbase.com${i18n.language.startsWith('zh') ? '/zh' : ''}/download`)
```

Comment:
I know, but I meant that instead of deciding on the language before the jump, the system language is automatically used at https://neuron.magickbase.com/download. This idea may not be related to this PR